### PR TITLE
Added the backend configuration for the terraform

### PR DESCRIPTION
**Pull request by athiruma**
Changed the env.conf to env.sh to export the variables from local env.
Added the terraform backend to s3

## Why?
terraform s3 backend configuration gives us the flexibility of storing the state file in s3_bucket. If further runs happen, it will directly **restore the state file from s3** and made changes to the current state.

## Pros
No need to manage the terraform state file.

### CODE REVIEW

File: cloudsensei/agg_lambda/lambda_function.py
Comment (krishvoor) [inline cloudsensei/agg_lambda/lambda_function.py:31]: This should be `Weekly Cloud Report: Long running instances in the Perf&Scale AWS Accounts`
Per line https://github.com/redhat-performance/cloud-governance/blob/main/cloudsensei/agg_lambda/lambda_function.py#L25

Comment (athiruma) [reply cloudsensei/agg_lambda/lambda_function.py:31]: Mentioned in the Subject, I think no need for every account.

Comment (krishvoor) [reply cloudsensei/agg_lambda/lambda_function.py:31]: Will this be a daily or weekly report?

Comment (athiruma) [reply cloudsensei/agg_lambda/lambda_function.py:31]: weekly report.

Comment (krishvoor) [reply cloudsensei/agg_lambda/lambda_function.py:31]: Cool, please add weekly, to help users distinguish it.

**Review by krishvoor — APPROVED** [review]
/lgtm

**Review by athiruma — COMMENTED** [review]
@krishvoor some changes are added. 
i.e env. conf to env.sh
ptlk

**Review by krishvoor — APPROVED** [review]

**Review by ebattat — COMMENTED** [review]
@athiruma, did you add rule against bucket to store data up to 3 mouthes, otherwise we will pay on old data?

**Comment by athiruma** [comment]
> @athiruma, did you add rule against bucket to store data up to 3 monthes, otherwise we will pay on old data?

I am using the cloud-governance s3 bucket.

**Comment by ebattat** [comment]
@athiruma I think we should add deletion rule for this bucket, what do you think keep only last 3 months ?

**Comment by athiruma** [comment]
> @athiruma I think we should add deletion rule for this bucket, what do you think keep only last 3 months ?

We can implement a lifecycle policy for the cloud-governance bucket. So it will move the data to s3 glacier after 3 months. WDYT?

**Comment by ebattat** [comment]
Why do you think we need this data in s3 glacier ?

**Comment by athiruma** [comment]
> Why do you think we need this data in s3 glacier ?

It is used for databackup.

**Comment by ebattat** [comment]
@athiruma Ok lets do it, pls let me know if you need my help

**Comment by krishvoor** [comment]
> > Why do you think we need this data in s3 glacier ?
> 
> It is used for databackup.

@athiruma I assume, we will store Terraform state file change in the S3 bucket, 
why would we need to retain it beyond 3 months?

**Comment by athiruma** [comment]
> > > Why do you think we need this data in s3 glacier ?
> > 
> > 
> > It is used for databackup.
> 
> @athiruma I assume, we will store Terraform state file change in the S3 bucket, why would we need to retain it beyond 3 months?

@krishvoor We always need to retain the terraform state file until we delete the services.

**Review by ebattat — APPROVED** [review]
Let me know when you add the s3 rule so I will merge it

**Comment by athiruma** [comment]
> /LGTM - Let me know when you add the s3 rule so I will merge it

Added, waiting for the @krishvoor reply and approval.

**Comment by krishvoor** [comment]
> > /LGTM - Let me know when you add the s3 rule so I will merge it
> 
> Added, waiting for the @krishvoor reply and approval.

/lgtm

**Comment by ebattat** [comment]
> krishvoor

Next time please approve it from the review changes window 
image

**Review by krishvoor — APPROVED** [review]
/LGTM

**Pull request merged**
